Examples/Examples/Models/Examples.swift

```diff
@@ -150,6 +150,9 @@ public struct Examples {
                 type: LineGradientExample.self),
         Example(title: "Animate a GeoJSON line",
                 description: "Update a map's style to animate a line over time.",
-                type: AnimateGeoJSONLineExample.self)
+                type: AnimateGeoJSONLineExample.self),
+        Example(title: "Use a map with SwiftUI",
```

Comment:
Should be "User a map & annotation with SwiftUI"

Reply:
Will update